src/components/application_manager/rpc_plugins/vehicle_info_plugin/include/vehicle_info_plugin/pending_resumption_handler.h

```diff
@@ -0,0 +1,64 @@
+#ifndef PENDING_RESUMPTION_HANDLER_H
```

Comment:
@mked-luxoft Wrong header guard

Reply:
@AByzhynar fixed in 23e9b6736a69f55e45d495d644b00c6437b37002